akka-cluster-tools/src/main/scala/akka/cluster/pubsub/DistributedPubSubMediator.scala

```diff
@@ -306,7 +306,8 @@ object DistributedPubSubMediator {
     def mkKey(path: ActorPath): String = path.toStringWithoutAddress
 
     trait TopicLike extends Actor {
-      import context.dispatcher
+      import scala.concurrent.ExecutionContext
```

Comment:
Let's keep the import "normal", at the top of the file along with all the other imports please

Reply:
(same for all other files i think)

Reply:
done with cf4fe9e91f